src/cli/utils.test.js

```diff
@@ -0,0 +1,85 @@
+import fs from 'fs';
+import path from 'path';
+import { utils } from './utils';
+import huronConfig from '../../test/config/huronConfig';
+import { dataStructure } from './huron-store';
+
+const parse = require('kss').parse;
+
+// Set huron config to our testing config
+// Disable for now, as we aren't using it yet
+/* eslint-disable */
+let mockData = dataStructure.set('huron', huronConfig);
+/* eslint-enable */
+
+/**
+ * Test for utils.normalizeSectionData()
+ */
+test('Ensures predictable data structure for KSS section data', () => {
```

Comment:
Use `it` instead of `test` they mean the same thing, but `it` is what we use elsewhere.